src/view.c

```diff
@@ -102,6 +102,21 @@ view_get_edge_snap_box(struct view *view, struct output *output,
 	return dst;
 }
 
+/*
+ * At present, a view can only 'enter' one output at a time, although the view
+ * may span multiple outputs. Ideally we would handle multiple outputs, but
+ * this method is the simplest form of what we want.
+ */
```

Comment:
Is this comment still valid, e.g. do we actually need to know if a view is on multiple outputs?